CMakeLists.txt

```diff
@@ -216,6 +216,13 @@ if (PROJECT_IS_TOP_LEVEL)
   # this is to find pip installed nvrtc .so
   set_target_properties(${NVFUSER_CODEGEN} PROPERTIES INSTALL_RPATH "$ORIGIN/../../nvidia/cuda_nvrtc/lib")
   install(TARGETS ${NVFUSER_CODEGEN} EXPORT NvfuserTargets DESTINATION lib)
+
+  add_custom_target(build_flatbuffer_config
```

Comment:
nitpick: maybe a simple comment here saying that we are keeping `fusion_cache_generated.h` for submodule build, where we don't have flatc available.

Don't have to do that here, since we already cleared the CI. But in some follow up clean-up PR.